### PR TITLE
feat(typography): remove arrow effect on <a>

### PR DESCRIPTION
**Pull request by bddylol**
(i hope I did this correctly) Removed the "arrow" when hovering over an HREF. Half the time it breaks, so I don't see the need for it.

### CODE REVIEW

File: src/styles/partials/_typography.css [hunk 1]
```diff
@@ -240,105 +240,6 @@ a {
...
-	}
-
-	a:after {
-		right: -10px;
-		bottom: 1px;
-		border: 5.5px solid transparent;
-	}
-}
-
-sup a {
```
Comment (rayhanadev) [inline src/styles/partials/_typography.css:311]: We actually need lines 311-320 so links in superscript (in effect, * references) have a different styling ;).

**Review by rayhanadev — CHANGES_REQUESTED** [review]
Thank you for this! I was actually planning styling links to how they look in their hover state, without the from highlight animation. If you could update the PR with that in mind it would be nice, otherwise I'd be willing to handle it. Thank youuuu! :D

File: src/styles/partials/_typography.css (hunk 1 above)
Comment (bddylol) [reply src/styles/partials/_typography.css:311]: ah, okay. wen't a little beyond the actual code. oopsies

**Review by rayhanadev — APPROVED** [review]
Approving this as a temporary solution until I think of something better! 👍

**Comment by rayhanadev** [comment]
Merged with main. Changes should be reflected on the production site soon.

I need to setup previews for pull requests 😅.

**Pull request merged**